docs/validators/setup/cli.md

```diff
@@ -64,30 +55,54 @@ Whenever you build or upgrade a node that includes a new hardfork, you must init
 
 ## Step 3. Start the node
 
-Run the following command to start a full (non-validator) Ronin node on the mainnet:
+To start a full (non-validator) Ronin node, run the following command:
+
+<Tabs groupId="networks">
+  <TabItem value="mainnet" label="Mainnet" default>
+
+  ```bash
+  ronin --http.api eth,net,web3,consortium --networkid 2020 --discovery.dns enrtree://AIGOFYDZH6BGVVALVJLRPHSOYJ434MPFVVQFXJDXHW5ZYORPTGKUI@nodes.roninchain.com --datadir /ronin/data \ --port 30303 --http --http.corsdomain '*' --http.addr 0.0.0.0 --http.port 8545 --http.vhosts '*' --ws --ws.addr 0.0.0.0 --ws.port 8546 --ws.origins '*' 
```

Comment:
The `/ronin/data \ --port 30303` seems incorrect. We use `\` so that we can break the line but the following part is still in the same command. So if you want all command in one line, remove `\`. If you want to break the line at that point, insert a newline after `\`

Reply:
@minh-bq I addressed your feedback, can you recheck?